lizmap/modules/lizmap/classes/lizmapWkt.class.php

```diff
@@ -18,6 +18,20 @@ class lizmapWkt
         'trimParens' => '/^\s*\(?(.*?)\)?\s*$/',
     );
 
+    /**
+     * Check if the WKT string is well formed.
+     *
+     * @param string $wkt A WKT string
+     *
+     * @return bool The WKT string is well formed
+     */
+    public static function check($wkt)
```

Comment:
Please use this new function in the following `public static function parse($wkt)`

Reply:
check returns `bool` not the `$matches` needed in `parse`.

Reply:
@mdouchin `check` is reused in `parse`.